crunchy/command.py

```diff
@@ -46,14 +48,14 @@ def run_command(self, parameters=None):
             command.extend(parameters)
 
         LOG.info("Running command %s", " ".join(command))
-        res = subprocess.run(
-            command, check=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE
-        )
+        res = subprocess.run(command, check=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 
         self.stdout = res.stdout.decode("utf-8").rstrip()
         self.stderr = res.stderr.decode("utf-8").rstrip()
         if res.returncode != 0:
-            LOG.critical(f"Call {command} exit with a non zero exit code", )
+            LOG.critical(
+                f"Call {command} exit with a non zero exit code",
+            )
```

Comment:
if you remove the comma this will be one line I think, do we want the comma there?

Reply:
👍